### PR TITLE
[EDGE] Support AITT as connection type. @open sesame 09/21 17:21

### PR DESCRIPTION
**Pull request by gichan-jang**
Edgesrc/sink support AITT as connection type.
Add unit test cases.

Signed-off-by: gichan <gichan2.jang@samsung.com>

**Self evaluation:**
1. Build test: [* ]Passed [ ]Failed [ ]Skipped
2. Run test: [ *]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

**Review by again4you — APPROVED** [review]
Good job!

**Review by myungjoo — APPROVED** [review]
I'm approving this only as a hotfix for 7.0M2.

Please write a commit for for-next branch, targetting nnstreamer 2.3.0 after 2.2.0 release that:
1. remove AITT condition from meson.build
2. remove #ifdef AITT from test case.
3. determine AITT existence at run-time of test case. You do not have any build-dependency on AITT from nnstreamer (or nnstreamer test cases).

File: meson_options.txt [hunk 1]
```diff
@@ -25,6 +25,7 @@ option('tvm-support', type: 'feature', value: 'auto')
 option('trix-engine-support', type: 'feature', value: 'auto')
 option('nnstreamer-edge-support', type: 'feature', value: 'auto')
 option('mxnet-support', type: 'feature', value: 'auto')
+option('aitt-support', type: 'feature', value: 'auto')
```
Comment (myungjoo) [inline meson_options.txt:28]: Add a comment or option-description to LINE28 that this is a workaround to be removed soon and users shouldn't depend on this option.

File: packaging/nnstreamer.spec [hunk 2]
```diff
@@ -229,6 +232,9 @@ BuildConflicts: libarmcl-release
 %if 0%{?edgetpu_support}
 BuildRequires:	pkgconfig(edgetpu)
 %endif
+%if 0%{?aitt_support}
+BuildRequires:  aitt-devel
+%endif
```
Comment (myungjoo) [inline packaging/nnstreamer.spec:237]: Add a comment that this is only required for unit testing.

**Comment by myungjoo** [comment]
@gichan-jang : merge the PR whenever you are ready. (I'm not merging this right away because I want to wait for you to read the comments above or #3917 )

File: packaging/nnstreamer.spec (hunk 2 above)
Comment (jaeyun-jung) [reply packaging/nnstreamer.spec:237]: Or, check nnstreamer_edge_support feature. (edgesrc/edgesink/tensor_query* all have dependency to nns-edge library.)

**Comment by gichan-jang** [comment]
> @gichan-jang : merge the PR whenever you are ready. (I'm not merging this right away because I want to wait for you to read the comments above or #3917 )

Checked, I will remove aitt dependency after this PR.

**Pull request merged**